openapi/leanix.integrationapi/types.bal

```diff
@@ -27,11 +27,11 @@ public type IdentifierSetTemplate record {
 public type RelationInboundProcessor record {
     *InboundProcessor;
     # The relation type
-    string 'type?;
+    string 'type;
```

Comment:
How option fields has become required fields is quite doubtful. Wondering whether someone manually made fields optional.

Reply:
As dicussed offline, we'll proceed with the updated code using latest tool